examples/multitenancy/appday2ops/web-app-chart/templates/basic-web-app-deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: web-app-deploy
+  labels:
+    app: web-app-deploy
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: web-app
+  template:
+    metadata:
+      labels:
+        app: web-app
+    spec:
+      containers:
+      - name: basic-web-app
+        image: eminalparslan/basic-web-app:0.3.0
```

Comment:
In the v2 chart, is this the image that will be changed?

Are both images public?

Reply:
Both charts refer to the same app image. The only difference is that the v2 chart contains a post-upgrade hook in `templates/post-upgrade-job.yaml`.

Yes, the app image should be public.

Reply:
Ok.